client/state/utils/reducer.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+import { UTILS_UNIQUEID_INCREMENT } from 'state/action-types';
+
+export function uniqueId( state = 1, action ) {
+	switch ( action.type ) {
+		case UTILS_UNIQUEID_INCREMENT:
+			state = state + 1;
+			break;
+	}
+
+	return state;
+}
+
+export default combineReducers( {
+	uniqueId
+} );
```

Comment:
> if we think we'll extend the reducers in the future

Exactly my intention, this file is `utils/reducer.js` and I think we may find other candidates to put here. everything thats memoizing or holding some internal state that needs to be put to the store